scripts/test-reviewer.mjs

```diff
@@ -170,7 +170,15 @@ for (let pattern in allTestsForPattern) {
 }
 
 const renderedIndex = mustache.render(indexTemplate, {
-  patterns: Object.keys(allTestsForPattern).map(pattern => ({ name: pattern, numberOfTests: allTestsForPattern[pattern].length }))
+  patterns: Object.keys(allTestsForPattern).map(pattern => {
+    const lastCommit = spawnSync('git', ['log', '-n1', '--oneline', path.join('.', 'tests', pattern)]).stdout.toString();
+    return {
+      name: pattern,
+      numberOfTests: allTestsForPattern[pattern].length,
+      commit: lastCommit.split(' ')[0],
+      commitDescription: lastCommit
+    }
+  })
```

Comment:
I guess we only needed this for pr-previewer, but maybe it's nice to leave in.

Reply:
I think it's still a general improvement to the page.